content/en/blog/_posts/2018-05-25-Airflow-Kubernetes-Operator.md

```diff
@@ -114,26 +112,90 @@ Use Travis or Jenkins to run unit and integration tests, bribe your favorite tea
+
+### ./scripts/ci/kubernetes/kube/deploy.sh
+
+Finally, we create a full airflow deployment on your cluster. This includes airflow configs, a postgres backend, the webserver + scheduler, and all necessary services between. One thing to note is that the role binding supplied is a cluster-admin, so if you do not have that level of permission on the cluster, you can modify this at scripts/ci/kubernetes/kube/airflow.yaml
+
+## Step 4: Log into your webserver
+
+Now that your airflow instance is running let's take a look at the UI! The UI lives in port 8080 of the airflow pod, so simply run 
+
+```
+WEB=$(kubectl get pods -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' | grep "airflow" | head -1)
+kubectl port-forward $WEB 8080:8080
+ ```
+
+ Now the airflow UI will exist on http://localhost:8080. To log in simply enter `airflow`/`airflow` and you should have full access to the airflow web UI.
+
+## Step 5: Upload a test document
+
+To modify/add your own DAGs, you can use `kubectl cp` to upload local files into the DAG folder of the Airflow scheduler. Airflow will then read the new DAG and automatically upload it to its system. The following command will upload any local file into the correct directory:
+
+`kubectl cp <local file> <namespace>/<pod>:/root/airflow/dags -c scheduler`
+
+## Step 6: Enjoy!
+
 # So when will I be able to use this?
 
-The Kubernetes Operator and Kubernetes Executor (article to come) have both been merged into the [1.10 release branch of Airflow](https://github.com/apache/incubator-airflow/tree/v1-10-test) (the executor in experimental mode). While this branch is still in the early stages, we hope to see it released for wide release in the next few months.
+ While this branch is still in the early stages, we hope to see it released for wide release in the next few months.
+
+# Get Involved
+
+This feature is just the beginning of multiple major efforts to improves Apache Airflow integration into Kubernetes. The Kubernetes Operator has been merged into the [1.10 release branch of Airflow](https://github.com/apache/incubator-airflow/tree/v1-10-test) (the executor in experimental mode), along with a fully k8s native scheduler called the Kubernetes Executor (article to come). These features are still in a stage where early adopters/contributers can have a huge influence on the future of these features.
+
+For those interested in joining these efforts, I'd recommend checkint out these steps: 
+
+ * Join the airflow-dev mailing list at dev@airflow.apache.org.
+ * File an issue in [Apache Airflow JIRA](https://issues.apache.org/jira/projects/AIRFLOW/issues/)
+ * Join our SIG-BIG-DATA meetings on Wednesdays at 10am PT.
+
+I'd like to thank the Apache Spark and Kubernetes communities, particularly Grant Nicholas, Ben Goldberg, Anirudh Ramanathan, Fokko Dreisprong, and Bolke de Bruin, for your awesome help on these features as well as our future efforts.
```

Comment:
The post starts off with "We..." taking the voice of the community. So, maybe reword to "Special thanks to ..."?